pkg/resourceinterpreter/customized/webhook/customized.go

```diff
@@ -70,7 +68,7 @@ func NewCustomizedInterpreter(informer genericmanager.SingleClusterInformerManag
 	}
 
 	cm.SetAuthenticationInfoResolver(authInfoResolver)
-	cm.SetServiceResolver(apiserver.NewClusterIPServiceResolver(serviceLister))
+	cm.SetServiceResolver(webhookutil.NewDefaultServiceResolver())
```

Comment:
Actually, before #2999 we used exactly the `webhookutil.NewDefaultServiceResolver()`, we need to figure out why changed to `apiserver.NewClusterIPServiceResolver(serviceLister)`.

Reply:
/hold
This essentially reverts #2999, and breaks the use case of running Karmada without coreDNS reported by @lxtywypc.